Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -1787,7 +1791,15 @@ private void createCaseData(ProgressIndicator progressIndicator) throws CaseActi
      */
     @Messages({
         "Case.progressMessage.openingCaseDatabase=Opening case database...",
-        "Case.exceptionMessage.couldNotOpenCaseDatabase=Failed to open case database."
+        "Case.exceptionMessage.couldNotOpenCaseDatabase=Failed to open case database.",
+        "# {0} - unsupported  version",
+        "# {1} - app name",
+        "# {2} - maximum supported major scheme version",
+        "Case.unsupportedSchemaVersionMessage=Unsupported DB schema version: {0}\n"
+        + "The highest supported DB schema version for this release of {1} is {2}.X",
+        "Case.databaseConnectionInfo.error.msg=Error accessing database server connection info. See Tools -> Options -> Multi-user.",
+        "Case.open.exception.multiUserCaseNotEnabled=Cannot open a multi-user case if multi-user cases are not enabled. "
+        + "See Tools, Options, Multi-user."
```

Comment:
I we are going to use arrows instead of commas to show a sequence of menu selections, we should be consistent. 

Also, the strings here are ever so slightly out-of-date: the options panel is now "Multi-User", not "Multi-user."